document-generator-company-report/src/main/java/uk/gov/companieshouse/document/generator/company/report/handler/CompanyReportDataHandler.java

```diff
@@ -89,14 +103,15 @@ private String getCompanyReportData(String companyNumber,  String requestId) thr
             requestId);
     }
 
-
     private String toJson(CompanyReport companyReport, String companyNumber,
                           String requestId) throws HandlerException {
 
         String reportToJson;
         ObjectMapper mapper = new ObjectMapper();
 
-        //TODO - Remove when definitly not required for any date mapping in company report
+
+//        //TODO - Remove when definitly not required for any date mapping in company report
+
 //        JavaTimeModule javaTimeModule = new JavaTimeModule();
 //        javaTimeModule.addDeserializer(LocalDate.class, new LocalDateDeserializer(DateTimeFormatter.ofPattern(DATE_TIME_FORMAT)));
 //        javaTimeModule.addSerializer(LocalDate.class, new LocalDateSerializer(DateTimeFormatter.ofPattern(DATE_TIME_FORMAT)));
```

Comment:
We could probably just remove this commented out code. As I'm pretty sure it's not needed now.